volpiano_display_utilities/text_volpiano_alignment.py

```diff
@@ -222,7 +222,7 @@ def align_text_and_volpiano(
     # Section volpiano to match text sections returned by syllabify_text
     # Split at clefs, barlines, and missing music markers, removing empty
     # sections created by the split.
-    volpiano_sections = re.split(r"([134]-*|6-{6}6-{3})", volpiano)
+    volpiano_sections = re.split(r"([134]-*|6-*6-*)", volpiano)
```

Comment:
Willing to trust you on this, as you've thought about it a bunch more than I have. But just to make sure: are we sure we want to match situations where there are no `-`s following the `6`s/`1`s/`3`s/`4`s? That is, should the regex be something along the lines of `[134]-+|6-+6-+`?

Reply:
It's a good question...I don't know the answer. I think here is another case that shows how ultimately we could turn this into kind of a spell checker for volpiano entry: Is it more likely that someone accidentally enters a `3` or a `4` with no following hyphen as a typo (the didn't mean to have a barline at all) or that they enter it meaning for it to be there but forget to put hyphens following? 

Maybe as a former indexer you have a sense? Or maybe @annamorphism does? To be honest, I really don't know, so if the general inclination is to require at least one hyphen to consider it actually a barline, I am happy to make the change!